specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/guestDiagnosticSettings_API.json

```diff
@@ -0,0 +1,580 @@
+                "operationId": "guestDiagnosticsSettings_Delete",                
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The diagnostic settings was successfully deleted."
+                    },
+                    "204": {
+                        "description": "The diagnostic settings does not exist. It may have already been deleted."
+                    },
+                    "default": {
+                        "description": "An error occurred and the diagnostic settings could not be deleted.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/microsoft.insights/guestDiagnosticSettings": {
+            "get": {
+                "description": "Get a list of all guest diagnostic settings in a subscription.",
+                "operationId": "guestDiagnosticsSettings_List",                
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsList"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the list of guest diagnostic settings could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/guestDiagnosticSettings": {
+            "get": {
+                "description": "Get a list of all guest diagnostic settings in a resource group.",
+                "operationId": "guestDiagnosticsSettings_ListByRG",                
```

Comment:
Please revert this to `"guestDiagnosticsSettings_ListByResourceGroup"`
This one didn't need to be changed.

Reply:
Done